script/client/spring/application.properties

```diff
@@ -0,0 +1,100 @@
+#
+#  Copyright 1999-2019 Seata.io Group.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+seata.enabled=true
+seata.application-id=test
+seata.tx-service-group=my_test_tx_group
+seata.client.rm-async-commit-buffer-limit=1000
+seata.client.rm-report-retry-count=5
+seata.client.rm-table-meta-check-enable=true
+seata.client.tm-commit-retry-count=5
+seata.client.tm-rollback-retry-count=5
+seata.client.rm.lock.lock-retry-interval=10
+seata.client.rm.lock.lock-retry-times=30
+seata.client.rm.lock.lock-retry-policy-branch-rollback-on-conflict=true
+seata.client.support.spring.datasource-autoproxy=true
```

Comment:
default false

Reply:
For file.conf,the default value is false.But for seata-spring-boot-starter,the value is true as default.